swap_meet/item.py

```diff
@@ -1,2 +1,35 @@
 class Item:
-    pass
\ No newline at end of file
+
+    def __init__(self, category="", condition=0):
+        # Each 'Item' will have an attribute named 'category' that is an empty string by default
+        self.category = category
+        self.condition = condition
+
+    def __str__(self):
+        """
+        String-ify an instance of 'Item' using 'str()'
+        """
+        return "Hello World!"
+    
+    def condition_description(self):
+
+        """
+        This function provides an item description based on self.condition
+        
+        * Condition 0 - 1 --> "Don't touch with bare hands"
+        * Condition 1 - 2 --> "Is it worth it? Probably not."
+        * Condition 2 - 3 --> "Meh, it's neutral."
+        * Condition 3 - 4 --> "It's a good find."
+        * Condition 4 - 5 --> "Top tier item! Must have."
```

Comment:
This is probably over documenting since any change to the code would require updating this documentation.